packages/orchestrator-ui-components/src/graphqlQueries/processListQuery.ts

```diff
@@ -59,3 +56,44 @@ export const GET_PROCESS_LIST_GRAPHQL_QUERY: TypedDocumentNode<
         }
     }
 `);
+
+export const GET_PROCESS_LIST_SUMMARY_GRAPHQL_QUERY = parse(gql`
```

Comment:
I would keep these queries in separate files. That's how the rest of the queries are setup. I agree they do relate to each other a lot. What do you think?

Reply:
I placed it here since this other query is a subset of the one above. We could also move this to a new file, but then I would like to introduce a subfolder to still group the almost-same queries better.

Reply:
Leaving it as-is for now. Lets discuss this later. Adding more similar queries like this we might want to revisit our placement of the types (types.ts vs close to the query)